tests/scalability/run_suite.sh

```diff
@@ -31,10 +31,11 @@ echo "jmeter do it!"
 ./jmeter/bin/jmeter -n  -t sync.jmx -Jworking_dir=./ -Jnode_binary=$(which node) -l ./report/cli_run.jtl -e -o ./report
 mv ./jmeter.log ./report/jmeter.log
 echo "Installing AWS CLI"
-sudo apt-get install unzip -y 
+sudo apt-get install unzip -y
 curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
 unzip awscliv2.zip
 sudo ./aws/install
 echo "Uploading logs and screenshots to ${S3_PATH}..."
 /usr/local/bin/aws s3 cp ./report "$S3_PATH" --recursive
-echo "FINISHED! "
\ No newline at end of file
+echo "FINISHED! "
+sudo shutdown -P +60
```

Comment:
Same here. The shutdown can be queued first.

Reply:
Done